front/app/containers/IdeasEditPage/IdeasEditForm.tsx

```diff
@@ -84,18 +79,49 @@ const IdeasEditForm = ({ ideaId }: Props) => {
     }
   }, [idea, granted, previousPathName, authUser]);
 
-  const initialFormData =
-    isNilOrError(idea) || !schema
-      ? null
-      : getFormValues(idea, schema, remoteImages, remoteFiles);
+  const getApiErrorMessage: ApiErrorGetter = useCallback(
+    (error) => {
+      return (
+        ideaFormMessages[
+          `api_error_${uiSchema?.options?.inputTerm}_${error}`
+        ] ||
+        ideaFormMessages[`api_error_${error}`] ||
+        ideaFormMessages[`api_error_invalid`]
+      );
+    },
+    [uiSchema]
+  );
+
+  const getAjvErrorMessage: AjvErrorGetter = useCallback(
+    (error) => {
+      return (
+        messages[
+          `ajv_error_${uiSchema?.options?.inputTerm}_${
+            getFieldNameFromPath(error.instancePath) ||
+            error?.params?.missingProperty
+          }_${error.keyword}`
+        ] ||
+        messages[
+          `ajv_error_${
+            getFieldNameFromPath(error.instancePath) ||
+            error?.params?.missingProperty
+          }_${error.keyword}`
+        ] ||
+        undefined
+      );
+    },
+    [uiSchema]
+  );
+
+  if (!idea) return null;
+
+  const projectId = idea.data.relationships.project.data.id;
+  const initialFormData = !schema
+    ? null
+    : getFormValues(idea, schema, remoteImages, remoteFiles);
```

Comment:
Minor: I think positive logic is easier to read but up to you

```suggestion
  const initialFormData = schema
    ? getFormValues(idea, schema, remoteImages, remoteFiles)
    : null;
```

Reply:
Agreed!